x-pack/plugins/apm/public/components/app/service_overview/service_overview_dependencies_table/index.tsx

```diff
@@ -173,22 +259,41 @@ export function ServiceOverviewDependenciesTable({ serviceName }: Props) {
       return callApmApi({
         endpoint: 'GET /api/apm/services/{serviceName}/dependencies',
         params: {
-          path: {
-            serviceName,
-          },
+          path: { serviceName },
+          query: { start, end, environment, numBuckets: 20 },
+        },
+      });
+    },
+    [start, end, serviceName, environment]
+  );
+
+  // Fetches previous period dependencies
+  const { data: previousPeriodData, status: previousPeriodStatus } = useFetcher(
+    (callApmApi) => {
+      if (!comparisonStart || !comparisonEnd) {
+        return;
+      }
+
+      return callApmApi({
+        endpoint: 'GET /api/apm/services/{serviceName}/dependencies',
+        params: {
+          path: { serviceName },
           query: {
-            start,
-            end,
+            start: comparisonStart,
+            end: comparisonEnd,
             environment,
             numBuckets: 20,
           },
         },
       });
     },
-    [start, end, serviceName, environment]
+    [comparisonStart, comparisonEnd, serviceName, environment]
```

Comment:
Shouldn't we add `comparisonEnabled` here?

Reply:
As agreed I'm going to do it on another PR.